doc/DESI_TARGET/targets-dr8.rst

```diff
@@ -0,0 +1,225 @@
+=======
+targets
+=======
+
+:Summary: DESI target selection files contain a single binary table containing
+    the targets in a (nested) HEALPixel. They contain the variables used by
+    target selection (*e.g.* fluxes), variables needed by fiber assignment (*e.g.*
+    RA, DEC), and variables needed for traceability (*e.g.* DESITARGET, TARGETID).
+:Naming Convention: ``IMAGING_VERSION/DESITARGET_VERSION/targets/SURVEY/RESOLVE/LAYER/targets-IMAGING_VERSION-hp-PIXNUM.fits``,
```

Comment:
Technically the naming convention is supposed to include only the filename.  The directory path is encoded in the actual path in desidatamodel.  Also, isn't there a leading catalog(s) directory?

Reply:
Ah, good point. I'll change this in the next 10 minutes to just include the filename. 

I think at some point I'll need to make a wider update so that the directory path is correctly encoded in the actual desidatamodel path. Let's leave that for later though. We probably need feedback from @sbailey on what to do about the fact that the directory structure for the mocks no longer aligns with the actual data.

Reply:
I would be fine with accepting this as-is as long as we don't forget to update that path in the future.